src/styles/index.ts

```diff
@@ -585,6 +588,10 @@ const styles = (theme: ThemeColors) =>
             borderRadius: variables.buttonBorderRadius,
         },
 
+        linkHover: {
+            color: theme.linkHover,
+        },
+
```

Comment:
Same here, we can use [StyleUtils.getColorStyle(theme.linkHover)](https://github.com/Expensify/App/blob/f8df4cf5a0068173a02201500c5672981e85fa15/src/styles/utils/index.ts#L966)